LiteCore/Support/Logging.cc

```diff
@@ -48,8 +49,11 @@ namespace litecore {
 
     LogLevel LogDomain::sCallbackMinLevel = LogLevel::Info;
     static LogDomain::Callback_t sCallback = defaultCallback;
+#if __ANDROID__
+    static bool sCallbackPreformatted = true;
```

Comment:
This is incorrect, now that you're formatting the string. It can cause crashes if the formatted string contains a `%` character.